tests/unit/shader_object.cpp

```diff
@@ -1812,6 +1812,11 @@ TEST_F(NegativeShaderObject, MissingCmdSetAlphaToOneEnableEXT) {
     m_errorMonitor->SetDesiredError("VUID-vkCmdDraw-None-07625");
 
     RETURN_IF_SKIP(InitBasicShaderObject());
```

Comment:
We have a easier way to do this now

```suggestion
    AddRequiredFeature(vkt::Feature::alphaToOne);
    RETURN_IF_SKIP(InitBasicShaderObject());
```